packages/website/src/components/Modals/RecruitmentModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import {
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from '@coderscamp/ui/components/Modal';
+import { Typography } from '@coderscamp/ui/components/Typography';
+
+import { forMentors } from './modal.data';
+import { ModalForm } from './ModalForm';
+
+type RecruitmentModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  modalText?: {
```

Comment:
💭 To be more DRY you can define an interface for modalText in modal.data.ts and only export it here.